docs/source/release_notes/release-notes.rst

```diff
@@ -57,13 +57,18 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2040>`__)
 
     *    - |improved|
+         - AtlasDB now supports batching of timestamp requests on the client-side; see :ref:`Timestamp Client Options <timestamp-client-config>` for details.
```

Comment:
it might be good to note the impact this has on lighter load (e.g., 2-4 threads).